src/spider/storage/MysqlStorage.cpp

```
@@ -0,0 +1,131 @@
+
+auto MySqlDataStorage::initialize() -> StorageErr {
+    return StorageErr{};
+}
+
+auto MySqlDataStorage::add_data(Data const& /*data*/) -> StorageErr {
+    return StorageErr{};
+}
+
+auto MySqlDataStorage::get_data(boost::uuids::uuid /*id*/, Data& /*data*/) -> StorageErr {
+    return StorageErr{};
+}
```

cppcheck

[performance] Variable 'm_id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)